pkr/driver/docker.py

```diff
@@ -48,9 +48,16 @@ class DockerDriver(AbstractDriver):
     def __init__(self, kard, *args, **kwargs):
         super().__init__(*args, kard=kard, **kwargs)
         self.metas = {"tag": None}
+        # If we have no overriding arugments use the configuration
+        # from the environment.
+        use_from_env = len(args) == 0 and len(kwargs) == 0
```

Comment:
this is not a good test. This value should be passed from the CLI with a --use-env-var for instance. We can discuss on the default value if you want.